src/components/generic(ui)/icon/Icon-interface.tsx

```diff
@@ -1,11 +1,13 @@
-import React, { ReactNode } from "react"
+import React, { ReactComponentElement, ReactNode } from "react"
+import { ReactI18NextChild } from "react-i18next"
 
 export interface IconProps {
     type?:string
     Img?:React.Component
     id?:string
     key?:string|number
-    text?: string
+    path:string
+    text?: string|ReactNode|React.Component|ReactI18NextChild|any
```

Comment:
pls dont use any, if you want to use union type. for example
type IconText = string | ReactNode and etc